client/src/components/autosave/filenameDialog.js

```diff
@@ -11,16 +11,19 @@ import {
   Tooltip,
 } from "@blueprintjs/core";
 
-@connect((state) => ({
-  idhash: state.config?.parameters?.["annotations-user-data-idhash"] ?? null,
-  annotations: state.annotations,
-  auth: state.config?.authentication,
-  userInfo: state.userInfo,
-  writableCategoriesEnabled: state.config?.parameters?.annotations ?? false,
-  writableGenesetsEnabled: !(
-    state.config?.parameters?.annotations_genesets_readonly ?? true
-  ),
-}))
+@connect((state) => {
+  console.log(state.config);
```

Comment:
please remove debugging code

Reply:
ideally, also revert to the previous lamba style (so we don't have an unnecessary diff in the PR)